mass_editing/__manifest__.py

```diff
@@ -15,9 +15,7 @@
     'license': 'GPL-3 or any later version',
     'summary': 'Mass Editing',
     'uninstall_hook': 'uninstall_hook',
-    'depends': [
-        'base',
-    ],
+    'depends': ['base', 'web'],
```

Comment:
why web?

Reply:
Yes Sir, "base, web" module are installed when we create db
but when i run with --test-enable "python3.5 odoo-bin --addons=addons,../projects/v12/MassEditing/ -d mass_editing_12_09 --http-port=1211 -i mass_editing --test-enable" then it gives an error 
`External ID not found in the system: web.assets_backend`

And somehow at the time Travis also failed, so for a check of Travis I've added "web" module in depends.

![mass_editing](https://user-images.githubusercontent.com/27808219/49936019-b95ca080-fef8-11e8-8ff7-b047cfb036e2.png)